src/components/Logo.js

```diff
@@ -0,0 +1,15 @@
+import { doc } from "prettier";
```

Comment:
Używasz tego? Co to jest?

Reply:
nie mam pojęcia skąd się to wzięło. Już usunięte